pkg/ui/react-app/src/thanos/pages/blocks/blocks.module.css

```diff
@@ -139,6 +139,22 @@
   box-shadow: 0 0 0 0.2rem rgba(0, 0, 0, 0.3) inset;
 }
 
+.downloadBtn {
```

Comment:
Instead of using custom css, use the `Button` component provided by `reactstrap` to be consistent. You can wrap the `Button` in `a` tag as you need it.

Reply:
Okay I have corrected this